### ISSUE TITLE
Variables described incorrectly as MSAgent.PlaneState instead of MSFSAgent.PlaneState

### ISSUE THREAD

**Issue by kaga-1011** [jamescl604/MSFSVoiceAttackPlugin#18]
in the list, it is incorrectly named all PlaneState variables

**Comment by KevinMicknick** [comment]
Have you been able to fix it and get it working? Are you on the Steam or Store version of MSFS 2020?

**Comment by kaga-1011** [comment]
> Have you been able to fix it and get it working? Are you on the Steam or Store version of MSFS 2020?

YES! (im in STEAM VERSION!) simply replace MSAGent.xxx.xxx with MSFSAgent.xxx.xxx. The only difference ist the extra FS after MS
